StackReviewSuggestedEditsRework.user.js

```diff
@@ -711,66 +711,45 @@
-                                                   // .querySelectorAll(".post-signature.grid--cell.fl0")];   // <-- answer
-                                                   // .querySelectorAll(".post-signature.owner.grid--cell")]; // <-- question
-                                               ];
-
-                                           const postUserCardContainer = createNewDiv();
-                                           // userCards.forEach(node => userCardDiv.appendChild(node));
-                                           postUserCardContainer.append(...userCards);
-
-                                           userCardsContainerAll.appendChild(postUserCardContainer);
-                                       }),
-
-                         404: () => {  // Status: Not Found
-                                  const messages = [
-                                                    "The original post is unavailable.",
-                                                    "User cards cannot be retrieved."
-                                                   ]
-                                  userCardsContainerAll.appendChild(missingCards(messages));
-                              },
-
-                         default : () => {  // Status: Don't know
-                                  const messages = [
-                                                    "Tried to fetch usercards, but",
-                                                    `Stack responsed with status: ${status}`
-                                                   ]
-                                  userCardsContainerAll.appendChild(missingCards(messages));
-                              }
-                     };
-
-                     (responseMap[status.toString()] || responseMap.default)(response);
-
-                     // Only necessary when keeping the Radio Button Box
-                     if (userConfig.options.radioVsButtons.moveRadioBox !== "Yes")
-                         userCardsContainerAll.style.width = adjustUserCardsWidth();
-                 })
-                .catch((error) => {
-                     // console.error(USERSCRIPTNAME + " - originalPostUserCards - error", error);
-                     const messages = [
-                                       "Something is blocking fetching user cards",
-                                       "Could be your ad-blocker. Check the console."
-                                      ];
-
-                     userCardsContainerAll.appendChild(missingCards(messages));
-              });
+            try {
+                const userCardRequest = await fetch(postlink);
+                const userCardResponse = await userCardRequest.text();
+                // https://developer.mozilla.org/en-US/docs/Web/API/DOMParser
+                // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
+                if (userCardRequest.status === 200) {
```

Comment:
not sure I like the `if...else if...else` structure over maps. @Scratle?

Reply:
@Oaphi this was updated in https://github.com/Scratle/fiddlings/pull/2/commits/70e3e7a68df11f33b74ba48d84ea75d519ab3aef to look better, does it still need a map?

Reply:
Methinks yes, I hold a particular dislike for the `else if` construct - but it's a non-blocking thing